pyproject.toml

```diff
@@ -19,8 +19,7 @@ Homepage = "https://github.com/armenzg/github-actions-app"
 [project.optional-dependencies]
 [tool.pdm.scripts]
 start = "flask run -p 5001"
-test = "pytest"
-coverage = "pytest --cov=src --cov-report=html && open htmlcov/index.html"
```

Comment:
I don't know how this worked in the past. It complains about the `&&`.
I'm okay with always generating the coverage.